pom.xml

```diff
@@ -123,23 +123,22 @@
   </pluginRepositories>
 
   <properties>
-    <amqp-10-starter.version>2.2.2</amqp-10-starter.version>
+    <amqp-10-starter.version>2.2.5</amqp-10-starter.version>
     <cxf-spring-boot-starter-jaxrs.version>3.3.4</cxf-spring-boot-starter-jaxrs.version>
-    <dekorate.version>0.11.2</dekorate.version>
+    <dekorate.version>0.11.4</dekorate.version>
     <infinispan-starter.version>2.1.8.Final</infinispan-starter.version>
-    <keycloak.version>9.0.0</keycloak.version>
+    <keycloak.version>9.0.2</keycloak.version>
     <narayana-starter.version>2.3.0</narayana-starter.version>
     <opentracing-spring-jaeger-web-starter.version>3.1.1</opentracing-spring-jaeger-web-starter.version>
-    <resteasy.version>3.9.0.Final</resteasy.version><!-- TODO upgrade before 2.2.5 release -->
-    <resteasy-spring-boot-starter.version>3.2.1.Final</resteasy-spring-boot-starter.version><!-- TODO upgrade before 2.2.5 release -->
+    <resteasy.version>3.9.0.Final</resteasy.version>
```

Comment:
This should be 3.9.3.SP1 https://github.com/resteasy/resteasy-spring-boot/blob/3.3.2.Final/resteasy-spring-boot-starter/pom.xml#L62

Reply:
It might be even better to remove it, because it's the same as in the starter.

Reply:
But we need it because we have the dependency:
```
 <dependency>
        <groupId>org.jboss.resteasy</groupId>
        <artifactId>resteasy-jaxrs</artifactId>
        <version>${resteasy.version}</version>
      </dependency>
```